docs/source/install.md

```diff
@@ -24,6 +24,11 @@ This requires compatible versions of `GDAL` and `numpy` from `conda-forge` for
 raw I/O support and `geopandas` and their dependencies for GeoDataFrame
 I/O support.
 
+To enable additional GDAL divers, you may need to install them directly:
+
+-   LIBKML: [libkml](https://anaconda.org/conda-forge/libkml)
```

Comment:
Based on the list of available plugins as conda packages (https://gdal.org/en/latest/download.html#conda) I would assume that KML is still built-in with libgdal-core?

Reply:
And the same for spatialite I assume

Reply:
It would still be good to mention this (the fact that some drivers need an additional install), but looking at that list, the only three vector drivers that get installed separately as plugin are `libgdal-arrow-parquet`, `libgdal-pg` and `libgdal-xls`, I think